File: framework/ui-elements/src/displays/components/TemplateStamp.js
import {withStyles} from 'alexandria-ui-elements/src/util/muiStylesCompat';
import AbstractTemplateStamp from "../../../gen/displays/components/AbstractTemplateStamp";
import TemplateStampNotifier from "../../../gen/displays/notifiers/TemplateStampNotifier";
import TemplateStampRequester from "../../../gen/displays/requesters/TemplateStampRequester";
import DisplayFactory from 'alexandria-ui-elements/src/displays/DisplayFactory';
import {withSnackbar} from "alexandria-ui-elements/src/util/notistackCompat";

const styles = theme => ({});

class TemplateStamp extends AbstractTemplateStamp {

	constructor(props) {
		super(props);
		this.notifier = new TemplateStampNotifier(this);
		this.requester = new TemplateStampRequester(this);
	};


}

export default withStyles(styles, { withTheme: true })(withSnackbar(TemplateStamp));
DisplayFactory.register("TemplateStamp", withStyles(styles, { withTheme: true })(withSnackbar(TemplateStamp)));

File: framework/ui/src/io/intino/alexandria/ui/displays/components/TemplateStamp.java
package io.intino.alexandria.ui.displays.components;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.TemplateStampNotifier;

public class TemplateStamp<DN extends TemplateStampNotifier, B extends Box> extends AbstractTemplateStamp<B> {

    public TemplateStamp(B box) {
        super(box);
    }

}